pkg/webhook/webhook.go

```diff
@@ -111,5 +113,26 @@ func register(manager manager.Manager, server *webhook.Server) error {
 				Decoder: manager.GetAdmissionDecoder(),
 			},
 		},
+	}, &admission.Webhook{
```

Comment:
Can we make this mutation webhook optional as I do not see it is needed for on prem use case ?

Reply:
It shouldn't hurt, right?

Reply:
I am a bit worried about the number of admission webhooks now which adds the overhead of launching deployments on cluster.